articles/multifactor-authentication/developer/step-up-with-acr.md

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
If trusting the current id token is ok, then that "transaction" belonging verification should have been done before (but the token still last a day)